system_tests/espresso_e2e_test.go

```diff
@@ -371,9 +374,34 @@ func TestEspressoE2E(t *testing.T) {
 		return balance4.Cmp((&big.Int{}).Add(transferAmount, transferAmount)) >= 0
 	})
 	Require(t, err)
+
+	// Now send the transaction for message pos 0, the message position 0 should have not been sent to espresso
+	// because its a genesis message which originates on L1
+	fetcher := func(pos arbutil.MessageIndex) ([]byte, error) {
+		msg, err := l2Node.ConsensusNode.TxStreamer.GetMessage(0)
```

Comment:
Does sending the genesis message really test that we don't resent messages that have been submitted to the l1? 

Wouldn't we have to essentially restart the batch poster at the end of the test with no state and see if it re-sends any message to espresso?

Reply:
As discussed in standup, this should be okay!